pkg/block/block_linux_test.go

```diff
@@ -4,6 +4,7 @@
 // See the COPYING file in the root project directory for full text.
 //
 
+//go:build linux
 // +build linux
```

Comment:
in a future PR, we should move this to the top of the file, to be consistent with all the other files. Same with other _test.go fles...